### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by TemesgenA**
## Features developed
- Created UI adaptable to different screen sizes using media queries
- GitHub Pages to deploy web pages
- Used JavaScript to manipulate DOM elements
- Recorded a video showing the details of the development process.
https://www.loom.com/share/eff365f22512403995c4539cbbce5217

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,13 +5,143 @@
...
+        </div>
+        <ul class="nav-menu">
+            <li><a class="nav-link" href="about.html">About</a></li>
+            <li><a class="nav-link" href="#main-program">Program</a></li>
+            <li><a class="nav-link" href="#join">Join</a> </li>
+            <li><a class="nav-link" href="#sponsor">Sponsor</a></li>
+            <li><a class="nav-link" href="#news">News</a></li>
+            <li class="nav-link campaign"><a href="#Campaign">WH Campaign</a></li>
+        </ul>
+    </nav>
```
Comment (elisha2kyakpo1) [inline index.html:39]: - [ ] kindly make the nav available on all the pages, I failed to navigate back to the main page from the about page.

no-nav

File: index.html [hunk 2]
```diff
@@ -5,13 +5,143 @@
...
+            </div>
+        </div>
+    </section>
+
+    <!-- Dynamic html loads here -->
+    <section id="featured-speakers">
+        <div class="section-title-style">
+            <h1>Featured Speakers</h1>
+            <div class="underline-section"></div>
+        </div>
```
Comment (elisha2kyakpo1) [inline index.html:114]: - [ ] The featured speaker's section images do not match the original, kindly fix it. Take a look at the photos below.
- [ ] The font size and font weight do not match the original, kindly work on that as well. 

### Yours

yours

### Original

original

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @TemesgenA, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The PR title and description looks professional 👍
- You have descriptive commit messages 👍
- The video is clear and the website is deployed 👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (TemesgenA) [reply index.html:39]: @elisha2kyakpo1
I have updated the live demo. It was pointing to the previous branch.
Now the issue is fixed.

File: index.html (hunk 2 above)
Comment (TemesgenA) [reply index.html:114]: @elisha2kyakpo1
- I have updated the live demo. It was pointing to the previous branch.
- Fix font size difference

**Review by AnselemOdims — APPROVED** [review]
### Hi  @TemesgenA 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**